crates/uv/tests/it/sync.rs

```diff
@@ -172,7 +172,7 @@ fn empty() -> Result<()> {
     )?;
 
     // Running `uv sync` should generate an empty lockfile.
-    uv_snapshot!(context.filters(), context.sync(), @r###"
+    uv_snapshot!(context.filters(), context.sync(), @r"
```

Comment:
Sorry for the churn here, these changed during testing and while the snapshot it the same the new version of insta uses different escapes.